spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -4,7 +4,7 @@ class TestAppGenerator < Rails::Generators::Base
   source_root '../../spec/test_app_templates'
 
   def install_engine
-    generate 'curation_concerns:install'
+    generate 'curation_concerns:install', '-f'
```

Comment:
To force overwrite the catalog_controller generated by hydra-head's installer.  Sufia does the same thing.